Makefile

```diff
@@ -12,6 +12,7 @@ OBJ = $(patsubst %,$(BUILDDIR)/%,$(_OBJ))
 
 
 $(BUILDDIR)/%.o: $(SRCDIR)/%.c $(DEPS)
+	@mkdir -p $(BUILDDIR)
```

Comment:
To nie działa pod windowsem. Mkdir nie ma flagi -p, a bez niej failuje jeśli katalog build już istnieje.